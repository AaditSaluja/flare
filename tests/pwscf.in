&CONTROL
    prefix = 'Al'
    calculation = 'scf'
    pseudo_dir = './test_files/pseudos'
    outdir = '.'
    tstress = .true.
    tprnfor = .true.
    disk_io = 'low'
    wf_collect = .false.
/ 
&SYSTEM
    ecutwfc = 10
    ecutrho = 80
    occupations = 'smearing'
    smearing = 'mp'
    degauss = 0.02
    ntyp = 1
nat = 4
    ibrav = 0
/ 
&ELECTRONS
    diagonalization = 'david'
    mixing_beta = 0.5
    conv_thr = 1e-07
/ 
&IONS
/ 
&CELL
/ 
K_POINTS {automatic}
 2 2 2  0 0 0
ATOMIC_SPECIES
  Al 26.9815385 Al.pz-vbc.UPF
CELL_PARAMETERS {angstrom}
3.9 0.0 0.0
0.0 3.9 0.0
0.0 0.0 3.9
ATOMIC_POSITIONS {angstrom}
Al -0.009633112297297939 0.0007024021249940832 -0.0016139992527199588
Al 0.15262256334224172 1.7096363872046911 1.8113886233139287
Al 1.9109729598639726 0.0038892355305986523 1.79679621754512
Al 1.7460375831206507 1.8857719688252057 0.09342917707343514

