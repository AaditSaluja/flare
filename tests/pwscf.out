
     Program PWSCF v.6.0 (svn rev. 13079) starts on 18Mar2019 at 14:40:18 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     2 processors
     R & G space division:  proc/nbgrp/npool/nimage =       2
     Waiting for input...
     Reading input from standard input
Warning: card &IONS ignored
Warning: card / ignored
Warning: card &CELL ignored
Warning: card / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
               file H.pbe-kjpaw.UPF: wavefunction(s)  1S renormalized

     gamma-point specific algorithms are used

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         358     358    178                 7121     7121    2519
     Max         359     359    179                 7122     7122    2522
     Sum         717     717    357                14243    14243    5041



     bravais-lattice index     =            0
     lattice parameter (alat)  =       9.4486  a.u.
     unit-cell volume          =     843.5418 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         2.00
     number of Kohn-Sham states=            1
     kinetic-energy cutoff     =      50.0000  Ry
     charge density cutoff     =     100.0000  Ry
     convergence threshold     =      1.0E-07
     mixing beta               =       0.5000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      =  SLA  PW   PBX  PBC ( 1  4  3  4 0 0)

     celldm(1)=   9.448631  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.000000 )  


     PseudoPot. # 1 for  H read from file:
     ./test_files/pseudos/H.pbe-kjpaw.UPF
     MD5 check sum: f0c5446d8b45f1c04d25babcc032f843
     Pseudo is Projector augmented-wave, Zval =  1.0
     Generated using "atomic" code by A. Dal Corso  (Quantum ESPRESSO distribution)
     Shape of augmentation charge: PSQ
     Using radial grid of  929 points,  2 beta functions with: 
                l(1) =   0
                l(2) =   0
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
        H              1.00     1.00000      H( 1.00)

      8 Sym. Ops. (no inversion) found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           H   tau(   1) = (   0.5070965   0.5000000   0.5000000  )
         2           H   tau(   2) = (   0.8929035   0.5000000   0.5000000  )

     number of k points=     1
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000

     Dense  grid:     7122 G-vectors     FFT dimensions: (  32,  32,  32)

     Estimated max dynamical RAM per process >       2.80Mb

     Estimated total allocated dynamical RAM >       5.60Mb

     Initial potential from superposition of free atoms
     Check: negative starting charge=   -0.000365

     starting charge    1.99998, renormalised to    2.00000

     negative rho (up, down):  3.652E-04 0.000E+00
     Starting wfc are    2 randomized atomic wfcs
     Checking if some PAW data can be deallocated... 

     total cpu time spent up to now is        0.1 secs

     Self-consistent Calculation

     iteration #  1     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  1.0

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  4.18E-04,  avg # of iterations =  1.0

     negative rho (up, down):  6.004E-05 0.000E+00

     total cpu time spent up to now is        0.1 secs

     total energy              =      -2.05428603 Ry
     Harris-Foulkes estimate   =      -2.05477198 Ry
     estimated scf accuracy    <       0.00760090 Ry

     iteration #  2     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  3.80E-04,  avg # of iterations =  1.0

     negative rho (up, down):  2.107E-05 0.000E+00

     total cpu time spent up to now is        0.2 secs

     total energy              =      -2.05454020 Ry
     Harris-Foulkes estimate   =      -2.05459720 Ry
     estimated scf accuracy    <       0.00156227 Ry

     iteration #  3     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  7.81E-05,  avg # of iterations =  1.0

     negative rho (up, down):  2.198E-05 0.000E+00

     total cpu time spent up to now is        0.2 secs

     total energy              =      -2.05463889 Ry
     Harris-Foulkes estimate   =      -2.05463341 Ry
     estimated scf accuracy    <       0.00000915 Ry

     iteration #  4     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  4.58E-07,  avg # of iterations =  3.0

     negative rho (up, down):  2.691E-05 0.000E+00

     total cpu time spent up to now is        0.2 secs

     total energy              =      -2.05465005 Ry
     Harris-Foulkes estimate   =      -2.05465150 Ry
     estimated scf accuracy    <       0.00000564 Ry

     iteration #  5     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  2.82E-07,  avg # of iterations =  2.0

     negative rho (up, down):  2.673E-05 0.000E+00

     total cpu time spent up to now is        0.2 secs

     total energy              =      -2.05465055 Ry
     Harris-Foulkes estimate   =      -2.05465077 Ry
     estimated scf accuracy    <       0.00000146 Ry

     iteration #  6     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  7.29E-08,  avg # of iterations =  1.0

     negative rho (up, down):  2.620E-05 0.000E+00

     total cpu time spent up to now is        0.2 secs

     total energy              =      -2.05465032 Ry
     Harris-Foulkes estimate   =      -2.05465059 Ry
     estimated scf accuracy    <       0.00000064 Ry

     iteration #  7     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  3.21E-08,  avg # of iterations =  2.0

     negative rho (up, down):  2.616E-05 0.000E+00

     total cpu time spent up to now is        0.2 secs

     total energy              =      -2.05465045 Ry
     Harris-Foulkes estimate   =      -2.05465049 Ry
     estimated scf accuracy    <       0.00000023 Ry

     iteration #  8     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  1.13E-08,  avg # of iterations =  1.0

     negative rho (up, down):  2.615E-05 0.000E+00

     total cpu time spent up to now is        0.2 secs

     total energy              =      -2.05465052 Ry
     Harris-Foulkes estimate   =      -2.05465046 Ry
     estimated scf accuracy    <       0.00000016 Ry

     iteration #  9     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  8.25E-09,  avg # of iterations =  1.0

     negative rho (up, down):  2.614E-05 0.000E+00

     total cpu time spent up to now is        0.2 secs

     total energy              =      -2.05465036 Ry
     Harris-Foulkes estimate   =      -2.05465053 Ry
     estimated scf accuracy    <       0.00000043 Ry

     iteration # 10     ecut=    50.00 Ry     beta=0.50
     Davidson diagonalization with overlap
     ethr =  8.25E-09,  avg # of iterations =  3.0

     negative rho (up, down):  2.616E-05 0.000E+00

     total cpu time spent up to now is        0.3 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (  2521 PWs)   bands (ev):

    -7.8458

     highest occupied level (ev):    -7.8458

!    total energy              =      -2.05465042 Ry
     Harris-Foulkes estimate   =      -2.05465044 Ry
     estimated scf accuracy    <       0.00000003 Ry

     total all-electron energy =        -2.054642 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =      -1.10873272 Ry
     hartree contribution      =       0.60409199 Ry
     xc contribution           =      -0.97857362 Ry
     ewald contribution        =      -0.57117669 Ry
     one-center paw contrib.   =      -0.00025939 Ry

     convergence has been achieved in  10 iterations

     Forces acting on atoms (Ry/au):

     atom    1 type  1   force =     0.07898029    0.00000000    0.00000000
     atom    2 type  1   force =    -0.07898029    0.00000000    0.00000000

     Total force =     0.111695     Total SCF correction =     0.000016

     Writing output data file pwscf.save

     init_run     :      0.04s CPU      0.05s WALL (       1 calls)
     electrons    :      0.13s CPU      0.15s WALL (       1 calls)
     forces       :      0.00s CPU      0.00s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
     potinit      :      0.01s CPU      0.02s WALL (       1 calls)

     Called by electrons:
     c_bands      :      0.01s CPU      0.02s WALL (      11 calls)
     sum_band     :      0.01s CPU      0.02s WALL (      11 calls)
     v_of_rho     :      0.08s CPU      0.09s WALL (      11 calls)
     newd         :      0.01s CPU      0.01s WALL (      11 calls)
     PAW_pot      :      0.01s CPU      0.01s WALL (      11 calls)
     mix_rho      :      0.01s CPU      0.01s WALL (      11 calls)

     Called by c_bands:
     init_us_2    :      0.00s CPU      0.00s WALL (      23 calls)
     regterg      :      0.01s CPU      0.01s WALL (      11 calls)

     Called by sum_band:
     sum_band:bec :      0.00s CPU      0.00s WALL (      11 calls)
     addusdens    :      0.00s CPU      0.01s WALL (      11 calls)

     Called by *egterg:
     h_psi        :      0.01s CPU      0.01s WALL (      29 calls)
     s_psi        :      0.00s CPU      0.00s WALL (      29 calls)
     g_psi        :      0.00s CPU      0.00s WALL (      17 calls)
     rdiaghg      :      0.00s CPU      0.00s WALL (      27 calls)

     Called by h_psi:
     h_psi:pot    :      0.01s CPU      0.01s WALL (      29 calls)
     h_psi:calbec :      0.00s CPU      0.00s WALL (      29 calls)
     vloc_psi     :      0.01s CPU      0.01s WALL (      29 calls)
     add_vuspsi   :      0.00s CPU      0.00s WALL (      29 calls)

     General routines
     calbec       :      0.00s CPU      0.00s WALL (      44 calls)
     fft          :      0.04s CPU      0.04s WALL (     147 calls)
     fftw         :      0.01s CPU      0.01s WALL (      69 calls)
     davcio       :      0.00s CPU      0.00s WALL (       1 calls)

     Parallel routines
     fft_scatter  :      0.03s CPU      0.03s WALL (     216 calls)

     PWSCF        :     0.27s CPU         0.31s WALL


   This run was terminated on:  14:40:19  18Mar2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
